main_utils.py

```diff
@@ -25,6 +25,10 @@ async def on_channel_create(self, channel):
         """
         EXTENSION 1 + 3: Staff notes for tickets! and button stripping.
         Minor Extension 2: Safe Community Support
+
+        This event works based on the ticket_users configuration.
+        Please ensure you add your ticket bot or approved
+         ticket creation users to the ticket users configuration.
```

Comment:
I still feel that "ticket creation users" and "ticket users configuration" could be better explained. 

Perhaps the former could be "ticket invoker" or "ticket creator" if relating to the creator of the ticket. 

Is the "Ticket users configuration" The panel config? If so or not, more specification would be nice

Reply:
GG on the first bit although

Reply:
At the end of the day, while the code is confusing, the general user is not going to look into the code and understand it in the way we do. If a programmer is desperately trying to work it out beyond the notes in the configuration file, I feel like that's all it needs